controllers/database_reconciler_helpers.go

```diff
@@ -202,13 +219,18 @@ func (r *DatabaseReconciler) handleSync(ctx context.Context, database *ndbv1alph
 		generatedReq, err := ndb_api.GenerateProvisioningRequest(ctx, ndbClient, databaseAdapter, reqData)
 		// generatedReq, err := ndb_api.GenerateProvisioningRequestt(ctx, ndbClient, database.Spec, reqData)
 		if err != nil {
-			log.Error(err, "Could not generate provisioning request, requeuing.")
+			errStatement := fmt.Sprintf("Could not generate provisioning request. Error: %s. Retrying", err.Error())
+			log.Error(err, errStatement)
+			r.recorder.Eventf(database, "Warning", "ProvisioningRequestGenerationFailed", errStatement)
 			return requeueOnErr(err)
 		}
+		r.recorder.Event(database, "Warning", "ProvisioningRequestGenerated", "Generated database provisiong request")
```

Comment:
Warning -> Normal

Reply:
As discussed, k8s supports only Normal and Warning as event types